modules/core/core-api/src/main/java/com/enonic/xp/app/ContentTypeParseMode.java

```diff
@@ -0,0 +1,32 @@
+package com.enonic.xp.app;
+
+public enum ContentTypeParseMode
```

Comment:
make it public internal static class of ApplicationWildcardMatcher and call it Mode